encoders/multimodal/TirgMultimodalEncoder/__init__.py

```diff
@@ -0,0 +1,78 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+import sys
+import pickle
+from typing import List
+
+import numpy as np
+
+from jina.executors.devices import TorchDevice
+from jina.excepts import PretrainedModelFileDoesNotExist
+from jina.executors.decorators import batching_multi_input, as_ndarray
+from jina.executors.encoders.multimodal import BaseMultiModalEncoder
+
+sys.path.append(".")
+from img_text_composition_models import TIRG
```

Comment:
Hide these `imports` inside functions so that it will avoid importing everything inside that file